curvefs/proto/topology.proto

```diff
@@ -57,6 +58,8 @@ enum OnlineState {
 // etcd struct defination
 message ClusterInfoData {
     required string clusterId = 1;
+    map<uint32, uint32> partitionIds = 2;
```

Comment:
where does this use

Reply:
persistent data to etcd need code date and decode when load.

Reply:
use a more meaningful name

Reply:
fixed